src/applications/claims-status/components/TravelClaimsSection.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import recordEvent from '@department-of-veterans-affairs/platform-monitoring/record-event';
+
+export default function TravelClaimsSection() {
+  const recordLinkClick = () =>
+    recordEvent({
+      event: 'nav-link-click',
+      'link-label': 'Review and file travel claims',
+      'link-destination': '/my-health/travel-pay/claims',
+    });
+
+  return (
+    <div className="vads-u-margin-y--4">
+      <h2>Your travel claims</h2>
+      <a
```

Comment:
Any reason we're not using the design system component for the link here? Is it just because the rest of the page isn't using the VaLink either? Just don't want us to get dinged for imposter components in the future & have to replace it anyway